app/assets/stylesheets/components/ui/_cookie.scss

```diff
@@ -15,26 +15,14 @@
 }
```

Comment:
The styles for `.cookie-form` can be removed now - I believe this is part of the position ing of the close icon as it is currently.